src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml.cs

```diff
@@ -15,24 +18,41 @@ namespace ApiPortVS.Views
     /// </summary>
     public partial class OptionsPageControl : UserControl
     {
+        private readonly IVsStatusbar _statusBar;
+
         private OptionsViewModel ViewModel { get { return DataContext as OptionsViewModel; } }
 
-        public OptionsPageControl(OptionsViewModel viewModel)
+        public OptionsPageControl(OptionsViewModel viewModel, IVsStatusbar statusBar)
         {
             InitializeComponent();
             DataContext = viewModel;
 
-            Loaded += async (s, e) => await viewModel.UpdateAsync().ConfigureAwait(false);
+            _statusBar = statusBar;
+
+            Loaded += async (s, e) => await UpdateModelAsync(false).ConfigureAwait(false);
```

Comment:
For me, this throws because another thread owns `ViewModel`.

Reply:
Might be because the UI is updated from a background thread? Maybe this will help: https://github.com/Microsoft/dotnet-apiport/blob/dev/src/ApiPort/ApiPort.VisualStudio.2017/VSThreadingService.cs#L21